packages/apps/reputation-oracle/server/src/common/guards/signature.auth.spec.ts

```diff
@@ -79,18 +78,34 @@ describe('SignatureAuthGuard', () => {
     });
 
     it('should throw unauthorized exception if signature is not verified', async () => {
-      (verifySignature as jest.Mock).mockReturnValue(false);
+      (verifySignature as jest.Mock).mockReturnValueOnce(false);
```

Comment:
Previous implementation of `verifySignature` couldn't return `false`, it could only throw if not verified, so probably we can remove this mock

Reply:
Agree on removing this mock but for a different reason :)